style.css

```diff
@@ -0,0 +1,58 @@
+* {
```

Comment:
- [ ] [REQUIRED] Please, make sure that your design is close to the figma design as possible.

- The hamburger icon is so big. Try to make it smaller.
- The icons doesn't show on the page because the server doesn't recognize root paths. In other words, you have used path like this `/icons/ angelist.png` where you should have used `./icons/angelist.png`.
- The hight of the whole section is small try to make it taller.

![image](https://user-images.githubusercontent.com/60979926/176608298-94b5e013-993a-4b20-8372-bb069e0f0239.png)